bundle/language_pt.properties

```diff
@@ -489,6 +489,7 @@ angal.lab.pleaseselectapatient
 angal.lab.positive                                                               = Positivo
 angal.lab.printreport                                                            = Imprimir relatório
 angal.lab.printtable                                                             = Imprimir tabela
+angal.lab.print																	 = Imprimir
```

Comment:
convertion space->tabs happening here

Reply:
Fixed with the push c9b6c019656dba1594baf288c36cf9d381f4130d